src/App.js

```diff
@@ -1,6 +1,4 @@
 import 'antd/dist/antd.css';
-import ButtonComponont from './Components/./Button/Button';
-
 function App() {
```

Comment:
I can't merge the app.js like this, please remove it from the PR and make sure you have the latest master version

Reply:
Yup done